xtab/org.eclipse.birt.report.item.crosstab.core/src/org/eclipse/birt/report/item/crosstab/core/util/CrosstabUtil.java

```diff
@@ -1116,6 +1116,10 @@ public static boolean validateBinding( ComputedColumnHandle column, String colum
 		{
 			columnName = "[^\\]]+";
 		}
+		else
+		{
```

Comment:
This use case should be invalid and return false. In fact, such expression should be blocked in syntax check in model, and should not hit engine runtime, so better fix is to enhance model syntax check to throw meaningful error.